src/Parser/Rogue/Assassination/Modules/Legendaries/DuskwalkersFootpads.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+import RESOURCE_TYPES from 'common/RESOURCE_TYPES';
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+
+// Equip: The remaining cooldown on Vendetta is reduced by 1 sec for every 65 Energy you expend.
+
+class DuskwalkersFootpads extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+    spellUsable: SpellUsable,
+      };
+
+  vendettaCDReduction = 1;
+  totalReduction = 0;
```

Comment:
make this a const above the class
``const VENDETTA_CDR = 1`` , it would probably make sense to call it ``const VENDETTA_CDR_PER_ENERGY = 1/65``, that would clean up your code down below